backend/app/lm/handlers.py

```diff
@@ -155,6 +152,12 @@ async def set_content(message=message):
             chat_completion_response,
         ).evaluate(session=self.session)
         # post-process the (request, response) pair
+
+        if  config.pii_removal == "replace":
+            chat_completion_data = chat_completion_response.choices[0].message.content
+#should I pass the entire mapping_list to replace_back, or is there a consistent position (like first or second) for the mapping of the response?
```

Comment:
actually you want to pass only the second, to be sure that no data from the examples may be leaked to another user